core/src/script/CGXP/plugins/Print.js

```diff
@@ -213,7 +237,7 @@ cgxp.plugins.Print = Ext.extend(gxp.plugins.Tool, {
 
     /** api: config[fields]
      *  ``Array``
-     *  Fields added in the print form.
+     *  Fields added in the print form, deprecated.
```

Comment:
What about emphasing that the parameter is deprecated, writing for instance ``(DEPRECATED)``?